sources/dataset.py

```diff
@@ -323,11 +327,11 @@ def build_glossary(self, nb_classes):
         nb_classes: integer
             Number of shape types (either 1, 2 or 3, warning if more)
         """
-        self.add_class(0, "square", [0, 10, 10], True)
+        self.add_class(0, "square", ShapeDataset.SQUARE_COLOR, True)
```

Comment:
You can write `self.SQUARE_COLOR` because it's a class attribute.

Reply:
Done